### ISSUE TITLE
Warning in React 16

### ISSUE THREAD

**Issue by balazsorban44** [dmtrKovalenko/material-ui-datetimepicker#8]
After updating to React 16, I get the following warning:

```
proxyConsole.js:54 Warning: React does not recognize the `timePickerDelay` prop on a DOM 
element. If you intentionally want it to appear in the DOM as a custom attribute, spell
it as lowercase `timepickerdelay` instead. If you accidentally passed it from a parent
component, remove it from the DOM element.
    in input (created by n)
    in div (created by n)
    in n (created by t)
    in span (created by t)
    in t (at NewBooking.jsx:408)
    in div (created by StepContent)
    in div (created by ExpandTransitionChild)
    in div (created by ExpandTransitionChild)
    in ExpandTransitionChild
    in div
    in TransitionGroup (created by ExpandTransition)
    in ExpandTransition (created by ExpandTransition)
    in ExpandTransition (created by StepContent)
    in div (created by StepContent)
    in StepContent (at NewBooking.jsx:405)
    in div (created by Step)
    in Step (at NewBooking.jsx:403)
    in div (created by Stepper)
    in Stepper (at NewBooking.jsx:336)
    in div (at NewBooking.jsx:335)
    in VerticalLinearStepper (at NewBooking.jsx:229)
    in div (created by DialogInline)
    in div (created by Paper)
    in Paper (created by DialogInline)
    in div (created by TransitionItem)
    in TransitionItem
    in div
    in TransitionGroup (created by DialogInline)
    in div (created by DialogInline)
    in DialogInline
```

**Comment by dmtrKovalenko** [comment]
Fixed in 0.2.0

**Comment by dmtrKovalenko** [comment]
@balazsorban44  Just updated this package for supporting react 16, bundle would also be much smaller. 
You can update for 1.0.0 (Also check breaking changes at the readme)
